synchronizer/l1_parallel_sync/l1_worker_etherman_test.go

```diff
@@ -27,7 +27,7 @@ func TestExploratoryWorker(t *testing.T) {
 		L1ChainID:                 1337,
 		ZkEVMAddr:                 common.HexToAddress("0x610178dA211FEF7D417bC0e6FeD39F05609AD788"),
```

Comment:
Still some addresses removed. You should keep all of them

Reply:
restored all address